### ISSUE TITLE
Function setBestParams( opt, dataset )

### ISSUE THREAD

**Issue by Jiankai-Sun** [amitshaked/resmatch#6]
In function FastNetwork:setBestParams( opt, dataset ) and function  AcrtNetwork:setBestParams(opt, dataset), there are lots of parameters. What do these parameters represents? I want to get the stereo information by using resmatch. I wonder whether these parameters are suitable for our  dataset, or what changes are need? Thank you!

>    self.params = {}
>    self.params.fm = opt.fm
>    self.params.decay=1e-4
>    self.params.at = 0
> 
> 
>    self.params.L1 = 0
>    self.params.tau1 = 0
>    self.params.pi1 = 4
>    self.params.pi2 = 55.72
>    self.params.sgm_i = 1
>    self.params.sgm_q1 = 3
>    self.params.sgm_q2 = 2.5
>    self.params.alpha1 = 1.5
>    self.params.tau_so = 0.02
>    self.params.blur_sigma = 7.74
>    self.params.blur_t = 5

**Comment by amitshaked** [comment]
- "fm" is the feature map size. 
- "decay" is the learning rate decay.
- "at" is deprecated.
- "L1" is the number of convolutional layers in the non residual networks.

The rest of the parameters are used at the refinement process, as described in this paper: https://arxiv.org/pdf/1510.05970.pdf

The parameters do change from dataset to dataset, see the differences for example between KITTI and MIDDLEBURY.

**Comment by Jiankai-Sun** [comment]
I use my images to test the provided pretrained [Resmatch-Kitti2012-Fast](https://www.mediafire.com/?9fvwnimchbtdb27) model. But I don't get the ideal result. I've already changed the input size parameters to 
>  height = 720
>  width = 1280

Is there any other parameters to change? E.g. the distance between two cameras? Which files are these parameters defined in? Thank you.

The inputs and output are shown below
The left image:
![1_10_left](https://user-images.githubusercontent.com/25477396/27079318-b6ac179c-5069-11e7-80a9-0c62adfcdfe1.png)
The right image
![1_10_right](https://user-images.githubusercontent.com/25477396/27079328-be730274-5069-11e7-8355-a9231e56f197.png)
The output:
![kitti_fast_1_10_right_pred](https://user-images.githubusercontent.com/25477396/27079295-aa2874fc-5069-11e7-92d4-c7faf5b5fc5c.png)

**Comment by amitshaked** [comment]
The distance between the cameras doesn't affect the disparity map, only the depth image you'd get. I'm afraid the Fast Resmatch model, because it's a shallower network, is more suited to autonomous car surroundings such as in the KITTI dataset, and applying it on rooms with low texture might actually give you a bad results. Try to train the Accurate Resmatch, or fine tune training on your room environment. There are no other parameters to change (I think the maximal disparity is OK even in your case). Hope it helps!

**Comment by Jiankai-Sun** [comment]
The problem has been solved! I made a mistake, the left image and the right image are reversed. Thank you for your advice!

**Comment by cjtang** [comment]
@JiankaiSun-SJTU-MVIG-training How can I test on my own image like you?  Could you help me about that ? 
Thank you very much !
